### ISSUE TITLE
Use of `:::` within package code

### ISSUE THREAD

**Issue by SCasanova** [SCasanova/f1dataR#138]
CRAN doesn't love the use of unexported functions: `Unexported object imported by a ':::' call: ‘reticulate:::conda_installed’`.
The comment is as follows:

```
It is typically a design mistake to use ::: in your code since the corresponding object has probably 
been kept internal for a good reason. 
Consider contacting the package [maintainer](http://127.0.0.1:39801/help/library/base/help/maintainer) 
if you feel the need to access the object for anything but mere inspection.
```

I will submit regardless but we might need to find alternatives if they reject us for this

**Comment by pbulsink** [comment]
Yeah, I know they're not gonna like it - if they don't accept it at all we'll either have to remove the whole helper function and replace with docs, or ask reticulate what they suggest instead.

**Comment by pbulsink** [comment]
Closed by #141
